packages/xo-web/src/common/intl/messages.js

```diff
@@ -2307,6 +2307,8 @@ const messages = {
   proxyLinkedBackups: 'Click to see linked backups',
   proxyNetworkDnsPlaceHolder: 'Default to: {dns}',
   proxyNetworkNetmaskPlaceHolder: 'Default to: {netmask}',
+  proxySrPredicateInfo:
+    'The selected SR should be at least connected to an HVM-capable host',
```

Comment:
```suggestion
    'The selected SR should be connected to at least one HVM-capable host',
```

Reply:
This message will change, see https://github.com/vatesfr/xen-orchestra/pull/4849/files#r390852176